generals.py

```diff
@@ -98,6 +98,8 @@ def get_updates(self):
 
             if msg[0] == "error_user_id":
                 raise ValueError("Already in game")
+           elif msg[0]== 'pre_game_start':
```

Comment:
Please could you add one more space before this line, otherwise it will throw an IndentationError? The number of spaces before each line should be a multiple of 4.